dataflows-composed/split-sentence/Makefile

```diff
@@ -1,6 +0,0 @@
-BASE_DIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST)))/../../)
-include $(BASE_DIR)/Makefile
-
-e2e:
```

Comment:
Looks like this is removing the things I added in #2 to get the examples in `dataflows-composed/` to run on CI